### PR TITLE
Display LV event duration

### PR DESCRIPTION
**Pull request by leandrocp**
While working on #9 I realized that instead of "display the status-online icon in place of the duration measurement when the liveview is connected" we could display the event duration which is helpful to debug slow events on development. WDYT?

### CODE REVIEW

File: lib/phoenix_profiler/live_view_listener.ex [hunk 1]
```diff
@@ -134,4 +149,8 @@ defmodule PhoenixProfiler.LiveViewListener do
 
     send(pid, {:navigation, view})
   end
+
+  defp notify_event_duration(pid, duration) do
+    send(pid, {:event_duration, duration})
```
Comment (leandrocp) [inline lib/phoenix_profiler/live_view_listener.ex:154]: At this moment that `duration` isn't persisted, it's just displayed on `metrics.duration.total` toolbar.

**Comment by mcrumm** [comment]
Hey @leandrocp, I like the idea but I want to make sure the values are very clear, so I do not think we should override the total duration. What if we added a third value, something like `latest_event_duration`? It could override the display in the toolbar.

**Comment by leandrocp** [comment]
> Hey @leandrocp, I like the idea but I want to make sure the values are very clear, so I do not think we should override the total duration. What if we added a third value, something like `latest_event_duration`? It could override the display in the toolbar.

Actually I agree, I started with a different value but wasn't sure if that was the right approach so I took the simplest idea. Anyway, I'm gonna update the PR 👍🏻

**Comment by leandrocp** [comment]
> Hey @leandrocp, I like the idea but I want to make sure the values are very clear, so I do not think we should override the total duration. What if we added a third value, something like `latest_event_duration`? It could override the display in the toolbar.

Done 👍🏻

**Comment by mcrumm** [comment]
This is awesome

**Pull request merged**